### PR TITLE
Black and white menu bar icon

### PR DESCRIPTION
**Pull request by BepsiKohler**
Addresses #26.

Adds the option to choose a minimalistic black/white menu bar icon from a Picker in the settings.
Works on both light and dark backgrounds.
If needed I can also include a vector file of the icon.

Love the project, hope I can help a bit! :)

### CODE REVIEW

**Comment by ThijmenDam** [comment]
Hi @BepsiKohler, really appreciate your input! Looks good. I'll make sure to take a detailed look at your PR this week. Thanks again!

**Comment by ThijmenDam** [comment]
Hi @BepsiKohler, looks solid.

Could you update the PR to include the vector file of the icon? After that, I'll do some testing myself, and if all is good I'll approve the PR and create a new release that includes your changes :-)

Thanks again for your work!

**Comment by BepsiKohler** [comment]
Hi @ThijmenDam,

the icon used to be in PNG format, however I've already updated the PR to use a vector file instead. The format might look a little bit weird, but thats just the way Xcode eats its vector symbols.

Also, if you are open to some more PRs before the new release, I made a fix for #25 as well as some tweaks to the UI.
But that's totally up to you :)

File: BarTranslate/DefaultSettings.swift [hunk 1]
```diff
@@ -14,9 +14,17 @@ enum TranslationProvider: String {
...
+  case original = "MenuIcon"
+  case minimal = "MenuIconMinimal"
+  
+  var id: String { self.rawValue }
+}
+
 struct DefaultSettings {
   
   static let translationProvider = TranslationProvider.google
+  static let menuBarIcon = MenuBarIcon.original
```
Comment (BepsiKohler) [inline BarTranslate/DefaultSettings.swift:27]: If you want you can set the new icon as default.

**Comment by ThijmenDam** [comment]
Hi @BepsiKohler, I have merged your PR into `master`. Thanks for your work, I really appreciate it. If you are still interested in publishing your PR for #25, I'd love to see it. If so, I'll make sure to include that PR into the next release. If not, I'll create a new release which includes the current PR. 

My apologies that approval took so long. That's on me!

**Pull request merged**